packages/core/test/dialog/dialogTests.tsx

```diff
@@ -31,6 +31,27 @@ describe("<Dialog>", () => {
         });
     });
 
+    it("renders contents to specified container correctly", () => {
+        const container = document.createElement("div");
+        document.body.appendChild(container);
+        mount(
+            <Dialog isOpen={true} container={container}>
+                {createDialogContents()}
+            </Dialog>,
+        );
+        [
+            Classes.DIALOG,
```

Comment:
just testing this one is sufficient here, since the test above asserts that if one exists then all of them do.